File: cases/taint/go/expression-positive/expression_positive.go
package dataflowbench

func dfb_source() int { // DFB-SOURCE: expression-input
	return 1
}

func dfb_sink(value int) {} // DFB-SINK: expression-sink

func run() {
	value := dfb_source()
	computed := (value * 3) + 7 // DFB-WITNESS: expression-computed
	dfb_sink(computed)
}

File: cases/taint/cpp/expression-positive/expression_positive.cpp
int dfb_source() { // DFB-SOURCE: expression-input
    return 1;
}

void dfb_sink(int value) {} // DFB-SINK: expression-sink

void run() {
    int value = dfb_source();
    int computed = (value * 3) + 7; // DFB-WITNESS: expression-computed
    dfb_sink(computed);
}
